optimize/util/optimize-commons/pom.xml

```diff
@@ -215,6 +209,10 @@
       <artifactId>auth</artifactId>
       <version>${awssdk.version}</version>
     </dependency>
+    <dependency>
+      <groupId>jakarta.ws.rs</groupId>
+      <artifactId>jakarta.ws.rs-api</artifactId>
+    </dependency>
```

Comment:
❓ Is this intentional?

Reply:
Yes, but I could have explained. The commons module declared the plugin one as a dependency and therefore pulled in this jakarta one. With the plugin module deleted, this now needs declaring here explicitly instead